workflow/scripts/prioritization/compile.py

```
@@ -55,18 +58,21 @@ def __init__(self, options):
-            vartype == "somatic.short.indels" or 
-            vartype == "long.indels" or 
-            vartype == "exitrons" or 
-            vartype == "altsplicing" or 
+        if (vartype == "somatic.snvs" or
+            vartype == "somatic.short.indels" or
+            vartype == "long.indels" or
+            vartype == "exitrons" or
+            vartype == "altsplicing" or
             vartype == "custom"):
 
             vars = variants.Variants(inputfile, options, vartype)
```

Ruff (0.15.17)

[error] Variable `vars` is shadowing a Python builtin

(A001)